### PR TITLE
Playwright_RGP_copied family member

### PR DESCRIPTION
**Pull request by Rinchan20**
family member is the brother. The test is skipped until housekeeping stuff is later fixed but can be un-skipped and ran, and should pass. Was checked against test.

### CODE REVIEW

**Comment by Rinchan20** [comment]
will mark as ready for review when https://github.com/broadinstitute/ddp-angular/pull/2180 is merged (create non-copied family member i.e. the grandfather)

**Comment by Rinchan20** [comment]
since I added more stuff - this is back to WIP status

**Comment by Rinchan20** [comment]
need a bit more work with this one - the flaky textarea is consistently being flaky again

File: playwright-e2e/dsm/pages/participant-page/rgp/family-member-tab.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { expect, Locator, Page } from '@playwright/test';
-import exp from 'constants';
 import { FamilyMember } from 'dsm/component/tabs/enums/familyMember-enum';
+import { text } from 'stream/consumers';
```
Comment (aweng98) [inline playwright-e2e/dsm/pages/participant-page/rgp/family-member-tab.ts]: not used.

**Review by aweng98 — APPROVED** [review]
LGTM

File: playwright-e2e/dsm/pages/participant-page/rgp/family-member-tab.ts (hunk 1 above)
Comment (Rinchan20) [reply playwright-e2e/dsm/pages/participant-page/rgp/family-member-tab.ts]: thanks for spotting it - got rid of the import

**Pull request merged**